server/game/cards/Time/RileTheMeek.js

```diff
@@ -11,8 +11,8 @@ class RileTheMeek extends Card {
         }
         let returnValue = {
             alwaysTriggers: true,
-            optional: true,
```

Comment:
i don't really understand what the different spots for optional mean, or what alwaysTriggers does